pkg/webhook/v1alpha2/component/mutating_handler.go

```diff
@@ -125,7 +127,9 @@ func (h *MutatingHandler) Mutate(obj *v1alpha2.Component) error {
 		// copy namespace/label/annotation to the workload and add workloadType label
 		workload.SetNamespace(obj.GetNamespace())
 		workload.SetLabels(util.MergeMap(obj.GetLabels(), map[string]string{WorkloadTypeLabel: workloadType}))
-		workload.SetAnnotations(obj.GetAnnotations())
+		// Add another annotation DefinitionAnnotation which can mark the name of WorkloadDefinition
+		workload.SetAnnotations(util.MergeMap(obj.GetAnnotations(), map[string]string{oam.DefinitionAnnotation: workloadType}))
```

Comment:
there is already a label to indicate the workloadType of the component. Where is this annotation with the same information used?

Reply:
Oh,yes, using the existing WorkloadTypeLabel is better.

Reply:
Will use the label, instead.